paddle/fluid/framework/ngraph_bridge.cc

```diff
@@ -104,11 +122,119 @@ static void BuildUnaryNode(
   SetOutputNode(op, "Out", out, ngb_node_map);
 }
 
+static void BuildMulNode(
```

Comment:
I see that all ngraph ops PR update the `paddle/fluid/framework/ngraph_bridge.cc`. It's not better due to:
- It will make `ngraph_bridge.cc` become much larger.
- The implementation of ngraph ops are not suitable in `fluid/framework`.

How about:
- make a new directory `fluid/operator/ngraph` (like `fluid/inference/tensorrt`)
- each ngraph op create a seperate file, such as `ngraph/mul_op.cc`